### PR TITLE
Create project and about sections

### PR DESCRIPTION
**Pull request by PolinaStamenova**
# Project and About - mobile version
**HTML:**
- Add "cards-project" section.
- Add "about-myself" section.

**CSS:**
- Add styling to project section.
- Add styling to about section.

**Linters:**
- Add webhint.
- Add stylint.
- Add lighthouse.

_Thank you for taking the time to review my PR!_ 🌵 😄

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,5 +60,141 @@ <h1 class="section-title">I’m Polina Glad to see you!</h1>
         </ul>
       </section>
     </main>
+    <section class="cards-project">
+      <div class="card">
```
Comment (smcommits) [inline index.html]: - [ ] **For listing projects, please use a `ul` tag to group them, and `li` per project.**

The `ul` and `li` tag order the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group. This helps in proper accessibility and optimization of the webpage.

File: index.html [hunk 2]
```diff
@@ -61,5 +60,141 @@ <h1 class="section-title">I’m Polina Glad to see you!</h1>
...
+            /></a>
+          </li>
+          <li>
+            <a href="#"
+              ><img src="images/icon-fingers.png" alt="Icon-Fingers"
+            /></a>
+          </li>
+        </ul>
+        <button type="button" class="btn-project">Get my resume</button>
+        <div class="skills-container">
```
Comment (smcommits) [inline index.html]: - [ ] **Please use `ul` and `li` tags to group all the elements in Skills section**

Again, as mentioned above, please try to use `ul` and `li` for semantic correctness whenever you are trying to list items. HTML has the ability to express lists of things, and it helps the Google robot, screen readers, and all manner of users who don't care solely about the presentation of the site understand your content better.

**Review by smcommits — CHANGES_REQUESTED** [review]
Hi @PolinaStamenova, 

### You have done an excellent job. The project appears to be fantastic. It reflects great efforts and good understanding. Nice use of flexbox and grid to align items. Your PR message is also descriptive. Also, the website appears to be exact clone of the design. Outstanding work. Keep it up!
### To make this project even better, there are some issues that you still need to work on. You are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


 ## Required Changes ♻️
_Please check the inline comments below._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (PolinaStamenova) [reply index.html]: Solved

File: index.html (hunk 1 above)
Comment (PolinaStamenova) [reply index.html]: Solved

**Comment by PolinaStamenova** [comment]
# Changes:

- Change _"div"_ to _"ul"_ in section with _class "butonns-languages"_.
- Change _"div"_ to _"ul"_ in section with _class "lg-dropdown"_.

_Thank you for taking time to review my PR_ 🌵 😄

File: index.html (hunk 1 above)
Comment (Temmarie) [inline index.html]: - [ ] As mentioned earlier by the previous code reviewer, the list of projects should be grouped using a `ul` tag to group them, and `li` per project as it orders the element semantically making it obvious to screen readers or assistive technologies to know they belong to the same group, this helps with proper accessibility. 
You have used `ul` - `li` to group the list of project technologies(e.g., "JavasScript", "HTML" and "CSS") but not the actual projects. Kindly make this change and submit it for review.

File: index.html [hunk 3]
```diff
@@ -61,5 +60,141 @@ <h1 class="section-title">I’m Polina Glad to see you!</h1>
...
```
Comment (Temmarie) [inline index.html]: - [ ] I see that you have grouped the list of technologies (e.g., "JavaScript", "HTML" and "CSS") under `Languages` skill using `ul` -`li`, good job there! However, the skills(languages, Frameworks, Skills) themselves should be grouped using `ul` - `li` as well.  This is what the previous reviewer meant by grouping all the elements in the Skills section.

**Review by Temmarie — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED! :recycle:

### Hi @PolinaStamenova    ! :wave: :wave:

#### You have done really well in implementing the previously required changes.
- You used proper GitHub Flow ✔️ 
- Your linter checks are all passing ✔️ 
- Your README and PR message are descriptive ✔️ 

 However, there are changes required for you to make in order for this project to be approved. 

> Keep in mind that your long hours of work are appreciated. However, the code review process is for improving the code you wrote, and not for criticizing it. :smile:

- - -

### **Required Changes** ♻️
_See the inline comments below for the required changes_


### Optional Changes 👌
Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better.
- N/A

_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

**_"Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."_**
- - - 
**Note that you still have 1 more review as written in the [Code Reviews Limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561), however, if you feel this review is unfair, you can request a second opinion using [this](https://airtable.com/shrQAqnBwek5a0O0s) form.**

__Kindly make the necessary changes and submit another code review request.__

**Comment by PolinaStamenova** [comment]
#Changes:

-Use "ul" in section with _class "cards-project-list"_.
-Change "div" to "ul" in section with _class "skills-container"_.

Thank you for taking time to review my PR 🌵 😄

**Review by ElijahTobs — APPROVED** [review]
# STATUS: APPROVED 🎉 🥇 
## Hi @PolinaStamenova , 

Good job making the changes required by the previous Code Reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

_Reviewed by_: __**Elijah Ayandokun**__

**Pull request merged**